### DISCUSSION TITLE
Line break syntax which follows commonMark convention is not working on v6.0.x

### DISCUSSION THREAD

**Discussion by khanhtc1202** [remarkjs/remark#996] [Q&A] [answered]
Due to the note on README, the react-markdown is 100% CommonMark compliant. We follow that note and use `\` to make a hard line break in the markdown source. The code worked perfectly at first, but somehow it changed its behavior to not support that convention anymore (I didn't update the react-markdown version).

Does anyone face the same problem? What should I do to keep `\` work in my case?

my project source (use react-markdown version v6.0.2): https://github.com/pipe-cd/pipecd/blob/master/web/src/components/application-detail-page/application-detail/index.tsx#L321-L323
codesanbox to test v6.0.2 react-markdown: https://codesandbox.io/s/react-markdown-basic-example-forked-v7ekqn

**Comment by wooorm** [comment, marked as answer]
Your problem is with the JavaScript language.

Your codesandbox doesn’t work because a `\` before character in a backticked string in JavaScript escapes it. Your example is an escaped line ending, which is just a line ending.

Use `\\` in a backticked JavaScript string, to get a slash.

You don’t have to do that in markdown files, because those are not JavaScript.

**Comment by khanhtc1202** [comment]
I get the point now, thanks for your help 🙏
